src/log.py

```diff
@@ -0,0 +1,55 @@
+"""
+log
+
+Author: Norma Chang
+"""
+import json
+import os.path
+import time
+
+
+class Log:
+    """
+    Logging class.
+
+    Logs data from vehicle.
+    """
+    def __init__(self, name):
+        self.file_name = ""
+        self._start_time = time.strftime('%b %d %Y %H:%M:%S', time.gmtime())
+        self._start_time = self._start_time.replace(" ", "").replace(":", "")
+
+        self.file_name = name + self._start_time + '.log'
+
+
+    def write_to_file(self, dictionary):
+        """
+        Write dictionary into text file.
+
+        :param: dictionary: The dictionary passed in from a class. 
```

Comment:
Fix this docstring.

Reply:
what needs to be modified? parameter? type? does it need more detail? does it need to be very specific about where the parameters are passed in from?